### DISCUSSION TITLE
2011/07/18/%E8%AE%BE%E8%AE%A1%E5%8F%AF%E8%B0%83%E8%8A%82%E5%A4%A7%E5%B0%8F%E7%9A%84%E7%A8%8B%E5%BA%8F%E7%95%8C%E9%9D%A2

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#341] [Announcements]
# 2011/07/18/%E8%AE%BE%E8%AE%A1%E5%8F%AF%E8%B0%83%E8%8A%82%E5%A4%A7%E5%B0%8F%E7%9A%84%E7%A8%8B%E5%BA%8F%E7%95%8C%E9%9D%A2

更新内容：  https://labview.qizhen.xyz/

https://blog.qizhen.xyz/2011/07/18/%E8%AE%BE%E8%AE%A1%E5%8F%AF%E8%B0%83%E8%8A%82%E5%A4%A7%E5%B0%8F%E7%9A%84%E7%A8%8B%E5%BA%8F%E7%95%8C%E9%9D%A2

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘磊磊
2011年09月1日 20:43 编辑
阮老师，请教您一个问题，我选择的数采卡是ni的16位数采卡PCI-6225，如果采集卡的采集范围是-5V到5V，而我实际信号电压范围是1-5V，这样对我采集到的数据精度的影响怎么评估？

回复
ruanqizhen
2011年09月2日 09:36 编辑
我不是这方面专家，但自己的想法是这样的：
采集卡是16位是它的分辨率：范围是10V，那么用电压表示的分辨率始终是 10 / 2^16 = 0.00015 V。
精度如何，板卡的文档中应该另有标明。

回复
Chien-Chi Wang
2011年07月22日 13:02 编辑
以下的範例當中有個Fit VI window to Largest Dec__ogtk.vi,我覺得也不錯使用
https://decibel.ni.com/content/docs/DOC-6503

回复

jianghaijun
2011年09月20日 00:19 编辑
我也想知道怎么改方便

回复
czhen
2011年07月19日 15:51 编辑
关注~
8.2以前研究过，没什么太好的办法，做出来效果也差强人意
后来也懒的搞了，直接固定分辨率了，投入产出比太低

回复
